composer.json

```diff
@@ -37,7 +37,8 @@
         "symfony/console": "^4.1",
         "symfony/debug": "^4.1",
         "symfony/process": "^4.1",
-        "symfony/var-dumper": "^4.1"
+        "symfony/var-dumper": "^4.1",
+        "vlucas/phpdotenv": "^3.0"
```

Comment:
Why have you added this buddy?

Reply:
See above

Reply:
@nunomaduro - I have put it back as it was as I am no longer touching the require object.